libraries/chain/transaction_metadata.cpp

```diff
@@ -16,6 +16,7 @@ recovery_keys_type transaction_metadata::recover_keys( const chain_id_type& chai
 
    g.unlock();
    // shared_keys_future not created or different chain_id
+   wlog( "recovering keys in recover_keys" );
```

Comment:
This seems like a `debug` log at best.

Reply:
Could change to debug. If developer called `start_key_recovery` as they should then you will never hit this line.

Reply:
Changed to `debug`.